.github/workflows/ci.yml

```diff
@@ -68,3 +68,30 @@ jobs:
 
       - name: Run tests
         run: ./vendor/bin/phpcs
+
+  static-analysis:
+    name: Static analysis
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php:
+          - '7.2'
+          - '7.3'
+          - '7.4'
+          - '8.0'
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.1.0
+        with:
+          php-version: ${{ matrix.php }}
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Download dependencies
+        run: composer update --no-interaction --prefer-dist --optimize-autoloader --prefer-stable
+
+      - name: Run PHPStan
+        run: ./vendor/bin/phpstan analyse --memory-limit 512M
```

Comment:
I suggest to add explicit phpstan level parameter here in order to always know what level we are on currently

Reply:
Level is defined in `phpstan.neon` file

Reply:
didn't noticed it. good!